spec/notifiers/job_performed_accept_notifier_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe JobPerformedAcceptNotifier, type: :mailer do
+  let(:mailer) { Struct.new(:deliver_later).new(nil) }
+  let(:job) { mock_model Job, owner: nil, accepted_applicant: nil }
+
+  it  'must work' do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.